src/components/App.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import LocationDetails from './LocationDetails';
+import getForecast from '../requests/getForecast';
+import ForecastSummaries from './ForecastSummaries';
+import ForecastDetails from './ForecastDetails';
+
+import '../styles/App.css';
+import SearchForm from './SearchForm';
+
+function App() {
+  const [forecasts, setForecasts] = useState([]);
+  const [location, setLocation] = useState({ city: "", country: "" });
+  const [selectedDate, setSelectedDate] = useState(0);
+  const [searchText, setSearchText] = useState("");
+  const selectedForecast = forecasts.find(
+    (forecast) => forecast.date === selectedDate,
+  );
+
+  useEffect(() => {
+    getForecast(setSelectedDate, setForecasts, setLocation);
+  }, []);
+
+  const handleForecastSelect = (date) => {
+    setSelectedDate(date);
+  };
+
+  const handleCitySearch = () => {
+    getForecast(
```

Comment:
The only thing stopping this call to getForecast from working is the content being passed into it. If you look at the getForecast component, it is expecting the props `setSelectedDate`, `setForecasts`, `setLocation` & `searchText`, but you are only passing in the first 3 of these. Also passing in `searchText` (in the right position in the call) will fix this